SlowCheetah.VisualStudio/Utilities/NugetHandler/NugetPackageHandlerBase.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.ComponentModelHost;
+using Microsoft.VisualStudio.Shell;
+using NuGet.VisualStudio;
+
+namespace SlowCheetah.VisualStudio
+{
+    public abstract class NugetPackageHandlerBase : INugetPackageHandler
+    {
+        protected IServiceProvider package;
+        protected const string pkgName = "SlowCheetah";
+
+        protected NugetPackageHandlerBase(IServiceProvider package)
+        {
+            this.package = package;
+        }
+
+        public bool IsSlowCheetahInstalled(EnvDTE.Project project)
```

Comment:
This is never used? Where are you checking if an appropriate version of SlowCheetah is installed?

Even then, it doesn't sound like this needs to be in this class. It can be within slowcheetah package itself.